src/ui/public/getting_started/registry.js

```diff
@@ -0,0 +1,78 @@
+import uuid from 'node-uuid';
+
```

Comment:
Chatted about this with @w33ble and @BigFunger out-of-PR. The decisions were:
* to allow multiple top messages, that way we don't have to worry about the single one getting clobbered. They will be concatenated and rendered just like the manage-and-monitor messages.
* replace the `getting_started/registry` module with two separate `ui/registry`s - one for top messages and one for manage-and-monitor messages.